base/linalg/symmetric.jl

```diff
@@ -32,13 +54,35 @@ end
 
 Construct a `Hermitian` matrix from the upper (if `uplo = :U`) or lower (if `uplo = :L`) triangle of `A`.
 
-**Example**
-
-```julia
-A = randn(10,10)
-Hupper = Hermitian(A)
-Hlower = Hermitian(A,:L)
-eigfact(Hupper)
+# Example
+
+```jldoctest
+julia> A = [1 0 2 0 3; 0 4 0 5 0; 6 0 7 0 8; 0 9 0 1 0; 2 0 3 0 4]
```

Comment:
probably better to use an example that's complex to make the distinction clearer?